pages/common/diff-pdf.md

```diff
@@ -0,0 +1,16 @@
+# diff-pdf
+
+> Tool for visually comparing two PDFs.
+> More information: <https://github.com/vslavik/diff-pdf>.
+
+- Compare PDFs, (return code 0 = no differences, return code 1 = PDFs differ):
```

Comment:
Is there no other indication?

```suggestion
- Compare PDFs using return codes (`0` = no difference, `1` = difference with PDFs)  :
```

Reply:
Quoted from the creator: "It takes two PDF files as arguments. By default, its only output is its return code, which is 0 if there are no differences and 1 if the two PDFs differ."

Reply:
Ok. I'll take their word :)

Whats your opinions on the suggestion?

Reply:
Well, i agree that the brackets or comma should go, but it doesn't "use" return codes to compare the PDFs

Reply:
Good point, my suggestion has been updated.